tests/testthat/test-new-model.R

```diff
@@ -146,3 +146,41 @@ test_that("new_model() supports `.path` containing a period [BBR-NWMD-010]", {
   mod <- new_model(fs::path_ext_remove(temp_ctl), "path with period")
   expect_true(fs::file_exists(temp_yaml))
 })
+
+test_that("new_model accepts .ctl file extension when passed [BBR-CLM-005]", {
+  clean_test_enviroment(create_rlg_models)
+  mod_ctl <- new_model(file.path(system.file("model", package = "bbr", mustWork = TRUE), "nonmem", "basic", "1.ctl"),
+    .description = "original acop model",
+    .tags = ORIG_TAGS,
+    .bbi_args = list(overwrite = TRUE, threads = 4)
+  )
+  expect_identical(mod_ctl$absolute_model_path, file.path(system.file("model", package = "bbr", mustWork = TRUE), "nonmem", "basic", "1"))
+  expect_identical(mod_ctl$model_type,"nonmem")
+  expect_identical(mod_ctl$yaml_md5,"6ccf206e167485b5adf29bc135197929")
+})
+
+test_that("new_model accepts .mod file extension when passed [BBR-CLM-006]", {
+  clean_test_enviroment(create_rlg_models)
+
+    #Make a copy of the .ctl to test for .mod and remove .ctl file
+    file.copy(file.path(system.file("model", package = "bbr", mustWork = TRUE),"nonmem", "basic","1.ctl"),
+    file.path(system.file("model", package = "bbr", mustWork = TRUE), "nonmem","basic", "1.mod" ))
+    file.remove(file.path(system.file("model", package = "bbr", mustWork = TRUE),"nonmem","basic","1.ctl"))
```

Comment:
* indentation is off
* use `file.rename` or `fs::file_move` to handle this in one step?
* I'd suggest setting up an `on.exit` handler for restoring the state rather than doing it at the end (line 181-84).  Even better would be to work within a temporary directory rather than `inst/`, but I know a lot of bbr's tests already do that, so that's fine.